src/AddTodoForm.js

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
 
-function AddTodoForm() {
-    return(
-        <form>
+function AddTodoForm(props) {
+
+    const handleAddTodo = (event) => {
+        event.preventDefault()
+        const todoTitle = event.target.title.value
+        props.onAddTodo(todoTitle)
+        event.target.title.value = ''
```

Comment:
@divinedouggy if you want to reset the whole form you can also just call 

`event.target.reset();`

event.target is your form and reset() is a method of your form that just resets all the fields